framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -2908,7 +2908,7 @@ private void stubWarnNotFound(NodeWithRange<?> astNode, String warning) {
    * @param warnIfNotFound whether to print warnings about types/members that were not found
    */
   private void stubWarnNotFound(NodeWithRange<?> astNode, String warning, boolean warnIfNotFound) {
-    if ((fileType.isCommandLine() || warnIfNotFound) || debugAnnotationFileParser) {
```

Comment:
Why are the parentheses needed?  I think they can be dispensed with.

Reply:
What parentheses?

Reply:
You are looking at the old line of code. The new line is (warnIfNotFound) || debugAnnotationFileParser)